util/loadtrace.py

```
@@ -0,0 +1,110 @@
+    return reasons
+
+stages = ['d','r','D','i','a','g','e','b','w','c']
+
+@lru_cache(maxsize=None)
+def DisAssemble(val):
+    if type(val) is str:
+        return val
+
+    hex_val = hex(val).lower()
+    command = f'echo "DASM({hex_val})" | spike-dasm'
+    asm = subprocess.run(command, shell=True, capture_output=True,
```

Ruff (0.14.8)

`subprocess` call with `shell=True` identified, security issue

(S602)